x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/index.tsx

```diff
@@ -55,15 +59,22 @@ export const StepRuleDescriptionComponent: React.FC<StepRuleDescriptionProps> =
 }) => {
   const kibana = useKibana();
   const [filterManager] = useState<FilterManager>(new FilterManager(kibana.services.uiSettings));
+  const [, siemJobs] = useSiemJobs(true);
 
   const keys = Object.keys(schema);
-  const listItems = keys.reduce(
-    (acc: ListItems[], key: string) => [
-      ...acc,
-      ...buildListItems(data, pick(key, schema), filterManager, indexPatterns),
-    ],
-    []
-  );
+  const listItems = keys.reduce((acc: ListItems[], key: string) => {
+    if (key === 'machineLearningJobId') {
```

Comment:
I added this branch here because I didn't want to add yet another optional argument to `buildListItems` that would only be used in one case.